pkg/store/proxy.go

```diff
@@ -273,10 +299,11 @@ func (s *ProxyStore) Series(originalRequest *storepb.SeriesRequest, srv storepb.
 		reqLogger = log.With(reqLogger, "request", originalRequest.String())
 	}
 
-	match, matchers, err := matchesExternalLabels(originalRequest.Matchers, s.selectorLabels)
```

Comment:
I'd suggest to pass in the `matcherConverter` instead:

```
match, matchers, err := matchesExternalLabels(originalRequest.Matchers, s.selectorLabels, s. matcherConverter)
```

Reply:
ack